src/app/components/menu/Menu.jsx

```diff
@@ -51,10 +68,14 @@ export default function TemporaryDrawer() {
 
   return (
     <div>
-      <Button onClick={toggleDrawer(true)}><MenuIcon fontSize='large' style={{ color: '#fff' }}/></Button>
+      <Button onClick={toggleDrawer(true)}>
+        <MenuIcon fontSize='large' style={{ color: '#fff' }} />
```

Comment:
jak już robimy style - to zróbmy spójnie wszędzie - czyli obiekt styles u góry i z niego skorzystajmy tutaj